src/assembly/jar.xml

```diff
@@ -0,0 +1,65 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+
+  <id>jar</id>
```

Comment:
Why do we need this file?
We are splitting the project in sub modules, it is better to have one jar per module.

I guess you are trying to reproduce the ant based build but it is not necessary to me.

Maven has already its clear way of distributing the artifacts.

Or maybe I am missing some part of the story

Reply:
Yes, that was the goal. Originally, we said we will reproduce ant build as much as possible. It has this fatjar in it.

I will do some revision on this PR.